package-lock.json

```diff
@@ -1,2880 +0,0 @@
-{
-  "name": "gratitudenu",
-  "version": "1.0.0",
-  "lockfileVersion": 1,
-  "requires": true,
-  "dependencies": {
-    "@babel/code-frame": {
-      "version": "7.12.11",
```

Comment:
Is there a reason we're deleting the package-lock files?

Reply:
I removed them from my commit but I didn't realize this would be deleting them. I wasn't sure if we have been committing package-lock files each time but I can push them if necessary.

Reply:
We've been committing them so far, so you can push them again